EIPS/eip-888.md

```diff
@@ -0,0 +1,178 @@
+2. Meritocracy -- Reputation is earned by recognition from the group. It cannot be claimed, purchased, or sold.
+3. Ethics -- The group can decrease an individual's reputation after bad behavior.
+
+Since the creation of Bitcoin in 2008, the vast majority of blockchain applications centered on buying and selling digital assets. While these use cases are substantial, digital assets need not be created with trading in mind. In fact, trading can be detrimental to community-based blockchain projects. This was evident in the pay-to-play dynamics of many EVM-based games and DAOs in 2021.
+
+A smart contract cannot directly imbue consent, meritocracy, and ethics into a community, but it can encourage those principles. In doing so, the standard set out below will hopefully unlock a diverse array of new use cases for tokens. 
+
+We considered a diverse array of use cases, though this may just be the beginning:
+- Voting weight in a DAO
+- Experience points in a decentralized game
+- Loyalty points for customers of a business
+
+This standard is influenced by the [ERC-20](./erc-20) and [ERC-721](./erc-721) token standards and takes cues from each in terms of its structure, style, and semantics. Neither, however, was created for fungible operator-managed token contracts such as EXP. Nor do existing proposals for non-tradable tokens meet the requirements of EXP use cases.
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Every ERC-888 compliant contract MUST implement the ERC888 and ERC165 interfaces:
+
+```
+// SPDX-License-Identifier: CC0
+
+pragma solidity ^0.8.0;
+
+/// @title ERC-888 EXP Token Standard
+/// @dev See https://eips.ethereum.org/EIPS/EIP-888
+///  Note: the ERC-165 identifier for this interface is ###ERC888###.
+///  Must initialize contracts with an `_operator` address that is not `address(0)`.
+///  Must initialize contracts assigning participation approval for both `_operator` and `address(0)`.
+interface IERC888 /* is ERC165 */ {
+
+    /// Emits when operator changes.
+    /// @dev MUST emit whenever operator changes.
+    event Appointment(address indexed _operator);
+
+    /// Emits when an address opts to participate.
+    /// @dev MUST emit whenever an address begins or ends participation.
+    ///  Transfers SHOULD NOT reset participation.
+    event Approval(address indexed _participant, bool _participation);
+
+    /// @notice Emits when operator transfers EXP to participating address. 
+    /// @dev MUST emit when EXP is created (`from` == 0), 
+    ///  destroyed (`to` == 0), or reallocated to another address.
+    ///  Exception: during contract creation, any amount of EXP
+    ///  MAY be created and assigned without emitting Transfer. 
```

Comment:
I wasn't sure why this exception is provided - not just that, but if the standard wishes for recipients of the token to opt in, wouldn't that be impossible inside the constructor?

Reply:
Good point. This is a remnant from an earlier implementation. Will remove the exception.